examples/tv-app/android/java/MediaPlaybackManager.cpp

```diff
@@ -92,40 +92,160 @@ uint64_t MediaPlaybackManager::HandleGetSeekRangeEnd()
 
 CHIP_ERROR MediaPlaybackManager::HandleGetActiveAudioTrack(AttributeValueEncoder & aEncoder)
 {
-    TrackType mActiveAudioTrack;
-    return aEncoder.Encode(mActiveAudioTrack);
+    return HandleGetActiveTrack(true, aEncoder);
+}
+
+CHIP_ERROR MediaPlaybackManager::HandleGetActiveTrack(bool audio, AttributeValueEncoder & aEncoder)
+{
+    Structs::TrackStruct::Type response;
+    Structs::TrackAttributesStruct::Type trackAttributes;
+    response.trackAttributes = Nullable<Structs::TrackAttributesStruct::Type>(trackAttributes);
+
+    jobject trackObj;
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    JNIEnv * env   = JniReferences::GetInstance().GetEnvForCurrentThread();
+
```

Comment:
Could you put the below JniLocalReferenceManager in the beginning of this HandleRecordProgram callback function so that we can automatically release local reference , when it is out of scope.
JNIEnv * env = JniReferences::GetInstance().GetEnvForCurrentThread();
VerifyOrReturn(env != nullptr, ChipLogError(Controller, "Could not get JNIEnv for current thread"));
JniLocalReferenceManager manager(env);

Reply:
fixed - thanks!